plugins/pagerduty/plugin.spec.yaml

```diff
@@ -68,6 +85,183 @@ types:
       - owner
       - read_only_user
       - user
+  incident_output:
+    id:
+      description: the id of the incident
+      type: string
+      example: PT4KHLK
```

Comment:
sensitive info?

Reply:
All of these examples were fake data or else taken from the pager duty api docs, but i'll update them to make them easier to look fake